### ISSUE TITLE
CipheyCore is set for archival

### ISSUE THREAD

**Issue by bee-san** [Ciphey/CipheyCore#19]
We are sunsetting CipheyCore for a full Python program. Please do not contribute to this repo :)

**Comment by KOLANICH** [comment]
It is still required by Ciphey.

**Comment by liudonghua123** [comment]
Yes, I could not install `Ciphey` due to depend on `cipheycore (>=0.3.2,<0.4.0)` which could not satisfied on python 3.11 or new.
